.github/workflows/ci.yml

```diff
@@ -8,6 +8,10 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
does it mean that if we have build wrt. one commit on master still in progress and we do a second commit, it will cancel the ongoing build and start a new build

Reply:
Yes.